### PR TITLE
Add WeTransfer upload provider

### PR DESCRIPTION
**Pull request by zeybek**
#### By submitting this pull request, I promise that:

- [x] I have read the [contribution guidelines](https://github.com/strapi/awesome-strapi/blob/master/CONTRIBUTING.md) thoroughly.
- [x] I ensure my submission follows each and every point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,22 +13,22 @@ Strapi is an open source Node.js Headless CMS to easily build customisable APIs
...
-	* [Nuxt.js](#Nuxt.js)
-	* [Angular](#Angular)
-* [Plugin & Providers](#PluginProviders)
-	* [Official Plugins](#OfficialPlugins)
-	* [Community Plugins](#CommunityPlugins)
-	* [Email Providers](#EmailProviders)
-	* [Upload Providers](#UploadProviders)
-* [Ecosystem](#Ecosystem)
-	* [SDKs](#SDKs)
+
```
Comment (derrickmehaffy) [inline README.md]: @zeybek can you revert the changes to the TOC. The extension for VSCode used to generate it is:
https://github.com/joffreykern/vscode-markdown-toc
https://marketplace.visualstudio.com/items?itemName=joffreykern.markdown-toc

Comment (zeybek) [reply README.md]: Fixed, sorry about that
